python/lsst/summit/utils/assessPeekExposure.py

```diff
@@ -0,0 +1,200 @@
+    petConfig = PeekExposureTask.ConfigClass()
+    pet = PeekExposureTask(config=petConfig, log=silent_logger, display=display)
+    bestEffort = BestEffortIsr(embargo=True)
+
+
+# retrieve best-effort-isr and run PET on it
+def doit(idx, row, doPlot):
+    exposureId = row["exposureId"]
+    dataId = {"instrument": "LATISS", "exposure": exposureId, "detector": 0}
+    exp = bestEffort.getExposure(dataId)
+    mode = "auto"
+    donutDiameter = None
+
+    # Incorrectly labeled as photo
+    if exposureId >= 2022101200584 and exposureId <= 2022101200588:
+        mode = "donut"
+        donutDiameter = 1500
+
+    # Incorrectly labeled as giant donut
+    if exposureId >= 2022101200589 and exposureId <= 2022101201247:
+        mode = "photo"
+
+    t0 = time.time()
+    result = pet.run(exp, mode=mode, donutDiameter=donutDiameter, doDisplay=doPlot)
+    t1 = time.time()
+    dt = t1 - t0
+
+    ax.text(
+        0.01,
+        0.97,
+        f"Runtime: {dt:.2f} s",
+        c="w",
+        transform=ax.transAxes,
+        fontsize=12,
+    )
+
+    inTag = row["finalTag"]
+    true_x = row["centroid_x"]
+    true_y = row["centroid_y"]
+    found_x = result.brightestCentroid.x
+    found_y = result.brightestCentroid.y
+
+    if np.isfinite(true_x):
+        n_x = (true_x - (result.binSize - 1) / 2) / result.binSize
+        n_y = (true_y - (result.binSize - 1) / 2) / result.binSize
+        if doPlot:
+            display.dot("o", n_x, n_y, ctype=afwDisplay.MAGENTA, size=20)
+
+        if np.isfinite(found_x):
+            dist = np.sqrt((true_x - found_x) ** 2 + (true_y - found_y) ** 2)
+            if dist < 20:
+                outTag = "<2"
+            elif dist < 100:
+                outTag = "<10"
+            else:
+                outTag = ">10"
+        else:
+            outTag = "noStar"
+    else:
+        if np.isfinite(found_x):
+            outTag = "noTruth"
+        else:
+            outTag = "nothing"
+
+    if doPlot:
+        if np.isfinite(result.psfShape.getIxx()):
+            distortion = ellipses.SeparableDistortionDeterminantRadius(result.psfShape)
+            fwhm = 2.35 * distortion.getDeterminantRadius() * 0.1
+        else:
+            fwhm = np.nan
+        display.show_colorbar(False)
+        ax.set_title(
+            f"{exposureId=}  {inTag=}  {outTag=}  {fwhm=:.2f} arcsec", color="w"
+        )
+        path = Path(args.plotdir)
+        path.mkdir(parents=True, exist_ok=True)
+        fn = f"test_pet_{idx:04d}_{exposureId:d}_{inTag}_{outTag}.png"
+        fig.savefig(path / fn)
+
+    return result, dt, inTag, outTag, idx, exposureId
+
+
+def main(args):
+    # Set up cache directory and register cleanup
+    defined, cache_dir = DatastoreCacheManager.set_fallback_cache_directory_if_unset()
+    if defined:
+        atexit.register(shutil.rmtree, cache_dir, ignore_errors=True)
+
+    # Loop through Merlin's curated data
+    path = Path("/sdf/data/rubin/u/jmeyers3/tickets/DM-36211/summit_extras/data")
```

Comment:
This is not ideal, but again, I'm hoping you have a good reason for it?

Reply:
Not anymore.  Now that DM-34703 is merged I can use $SUMMIT_EXTRAS_DIR for at least a default value.